CHANGELOG.md

```diff
@@ -35,6 +35,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
+## [Unreleased Osmosis fork]()
```

Comment:
Are we committing to maintaining a CL in our fork now? If so, let's' be sure to not half-ass it.

While on the topic, what's our stance on upstreaming changes to the core repo?